### DISCUSSION TITLE
__INITIALIZER.unity' is missing or no valid script is attached.

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#639] [Issue Report (Legacy)]
월드 1.2.0

다음과 같이 테스트용 C# 스크립트를 만들어 사용중이였습니다 
![image](https://user-images.githubusercontent.com/50188795/164031266-32112026-669b-44b9-a842-241e30eb08be.png)
퍼블리싱 테스트를 위해 C#스크립트를 제거 하고 퍼블리싱을 했더니 다음과 같이 에러가 났습니다.
![image](https://user-images.githubusercontent.com/50188795/164031389-427747bc-9354-4aa1-8add-f7ba86450288.png)
![image](https://user-images.githubusercontent.com/50188795/164031095-b2d5a7e1-9a5e-41ef-bbfb-6698bca822ce.png)
어떻게 해야 할까요?

로그 파일 첨부 합니다 
[com.smilejsu.test001_0.1_20220419.txt](https://github.com/naverz/zepeto-studio-kor/files/8514026/com.smilejsu.test001_0.1_20220419.txt)

**Comment by smilejsu82** [comment]
#476를 참고 했습니다 

Library, obj 등 캐시 폴더를 지우고 재시작 
퍼블리시 성공 했습니다

**Reply by x4cf** [reply]
좋아요 :D
